src/commands/test.js

```diff
@@ -6,60 +6,142 @@ const spawn = require("cross-spawn").spawn;
 const path = require("path");
 const CWD = process.cwd();
 const chokidar = require("chokidar");
+const logger = require("../lib/logger");
 
-function runMocha(options, callback) {
-  const testPath = `${CWD}/test/**/*.test.js`;
-  const helperPath = path.join(__dirname, "..", "lib", "testHelperMocha.js");
+const MOCHA_PATH = path.join(__dirname, "..", "..", "node_modules", ".bin", "mocha");
+const MOCHA_PATH_COVERAGE = path.join(__dirname, "..", "..", "node_modules", ".bin", "_mocha");
+const MOCHA_ENV = { ...process.env, ...{ NODE_PATH: CWD, NODE_ENV: "test" }};
+const MOCHA_TEST_PATH = `${CWD}/test/**/*.test.js`;
+const MOCHA_HELPER_PATH = path.join(__dirname, "..", "lib", "testHelperMocha.js");
+const BABEL_NODE_PATH = path.join(__dirname, "..", "..", "node_modules", ".bin", "babel-node");
+const BABEL_ISTANBUL_PATH = path.join(__dirname, "..", "..", "node_modules", "babel-istanbul", "lib", "cli.js");
+const INSPECTOR_PATH = path.join(__dirname, "..", "..", "node_modules", ".bin", "node-inspector");
```

Comment:
Since the node_modules path is used a lot, you can even separate this out if you think it will be cleaner:

```
const NODE_MODULES = path.join(__dirname, "..", "..", "node_modules");
const MOCHA_PATH = path.join(NODE_MODULES, ".bin", "mocha")
const MOCHA_PATH_COVERAGE = path.join(NODE_MODULES, ".bin", "_mocha");
const BABEL_NODE_PATH = path.join(NODE_MODULES, ".bin", "babel-node");
...
```